src/controllers/initgamecontroller.js

```diff
@@ -59,20 +72,26 @@ class InitGameController extends Controller
     registerEvent()
     {
         // toggles open/close menu
-        $("#menu-toggle").click(function (e) {
+        $("#sidebar-left-toggle").click(function (e) {
             e.stopPropagation();
 
-            $("#overlay-menu").width(250);
+            $("#sidebar-left").width(250);
             $(".wrapper").css({marginLeft: 250, opacity: .3});
         });
 
+        // find way to merge these two events
```

Comment:
Turn the internals into a closure and call the function in both events with an anchor tag (`#sidebar-left` or `#sidebar-right`) and then the required css changes as an object. So a function like `function (anchor, css)`.